flaky-tests/frameworks/pytest.md

```diff
@@ -46,24 +43,30 @@ would produce output that looks like this:
      ...
 </testsuites>
 ```
-
-The suite name [can be configured](https://docs.pytest.org/en/7.2.x/how-to/output.html) in the `pytest.ini` or [similar config file](https://docs.pytest.org/en/8.1.x/reference/customize.html#pytest-ini).
+The suite name [can be configured](https://docs.pytest.org/en/7.2.x/how-to/output.html) in the `pytest.ini` or [similar config file](https://docs.pytest.org/en/8.1.x/reference/customize.html#pytest-ini). 
 
 ```ini
 [pytest]
 junit_suite_name = my_suite
 ```
-
-Configuring other XML output values is not currently supported, but experimental options are [available](https://docs.pytest.org/en/7.2.x/how-to/output.html#record-xml-attribute).  To include the test filenames in the XML output, use the `-o junit_family=xunit1` option.
+Configuring other XML output values is not currently supported, but experimental options are [available](https://docs.pytest.org/en/7.2.x/how-to/output.html#record-xml-attribute).  To include the test filenames in the XML output, use the `-o junit_family=xunit1` option.
 
 ```shell
 pytest --junitxml=output/path.xml -o junit_family=xunit1
 ```
 
-## Further Configuration
 
-pytest is only one of several popular testing frameworks for Python. You might also want to look at
+
+
+## Further Information
+See an example of running **pytest** in a GitHub action [here](https://github.com/trunk-io/flake-factory/blob/main/.github/workflows/python-tests.yaml#L34).
+
+**pytest** is only one of several popular testing frameworks for Python. You might also want to look at
 
 * [Behave](https://behave.readthedocs.io/en/latest/)
```

Comment:
Same here with external links, should we link to outside docs? Is this leftover from the SEO pages?

Reply:
fixed

Reply:
removed.